include/swift/Reflection/ReflectionContext.h

```diff
@@ -30,6 +30,7 @@
 #include "swift/Reflection/TypeRef.h"
 #include "swift/Reflection/TypeRefBuilder.h"
 #include "swift/Runtime/Unreachable.h"
+#include "../../../stdlib/public/runtime/ImageInspectionELF.h"
```

Comment:
We could move the `MetadataSections` definition to a public `swift/Runtime` header, so you don't need to reach across the source tree like this.